index.js

```diff
@@ -15,10 +15,26 @@ const manageRecipes = async () => {
     const dbConnection = await mongoose.connect(MONGODB_URI);
     console.log(`Connected to the database: "${dbConnection.connection.name}"`);
 
+    
+    
     // Before adding any recipes to the database, let's remove all existing ones
     await Recipe.deleteMany();
 
     // Run your code here, after you have insured that the connection was made
+    const bolonhesa = await Recipe.create({title:'Bolonhesa', cuisine:'Italiana'})
+    
+    await Recipe.insertMany(data);
+    const recipesArray = await Recipe.find();
+
+    recipesArray.forEach(recipe => {
+      console.log(recipe.title)
```

Comment:
Another way of doing is:

´´´
    const manyRecipes = await Recipe.insertMany(data);
    manyRecipes.forEach((recipe) =>
      console.log(`New recipe added: ${recipe.title}`)
    );
´´´